README.md

```diff
@@ -1,2 +1,87 @@
+
+## Accountability
+
+1. Much like _Core Team_, minutes and reports from the _Dev Team_, and its various sub-groups will be available to view by the general public. Sensitive data, such as budgeting, may be exempt from this.
+2. The _Dev Team_ will submit regular progress reports to the Trustees.
+3. As well as regular volunteers, the _Dev Team_ may contain trustees and members of the _Core Team_, however volunteers should bear in mind any potential conflicts of interest that may arise as a result of this.
+4. Members of the _Dev Team_ should abstain from financial decisions where they are directly involved with the _project group_ of concern. In an event when all members of the _Dev Team_ have to abstain, the decision should be passed to the _Core Team_ or Trustees.
+5. The _Dev Team_ is responsible for the oversight of the various project groups within Student Robotics. The _project groups_ should report their progress regularly to the Dev Team.
+
+## Relationship to the Core Team
+
+- The _Dev Team_ is independent from the _Core Team_, but will work closely with them in certain situations.
+- The _Core Team_ will also communicate priorities for projects to be completed by the next competition cycle. In the event of a Kit Deployment, the _Core Team_ is responsible for budgeting, manufacture, deployment and testing.
+
+## Getting Things Done
+
+For new projects, there is a process to follow to ensure they receive sufficient review and testing before they are implemented. This is shown in the examples below.
+
+For each _project group_ it is recommended that a “Project” is opened and actively maintained with high-level progress updates in the `srobo` GitHub - even the “nice to haves” - if there are good ideas being worked on which get parked here, at least there will be some documentation to pick up on.
+
+Examples of what may be in scope for SR2020: Python 3, new brain board, new IDE.
+
+### Project of significant impact: New brain board
+
+- A (group of) volunteer(s) decide(s) they’d like to work on finding a new single board computer which can serve as the brain board, and port/upgrade software to work on it.
+- New / Updated ticket in srobo/dev-tasks.
+- The volunteers develop a brief specification, including information like:
+    - Functional requirements (e.g. compatible with the rest of the kit, can be powered from Power Board, runs our API, camera isn’t made slower, form factor)
+    - Scope (e.g. excludes any changes to modular kit or API, independent of Python 3 project)
+    - Deliverables (e.g. Board, Software, Documentation for students and volunteers)
+    - Testing requirements (e.g. is demonstrated to run a defined piece of software on a test robot without fault, with a certain response time for libkoki markers, etc.)
+- The _Dev Team_ may also define some limits, such as:
+    - Cut-off date for deployment at SR2020
```

Comment:
The core team should define the cut off date for deployment. The dev team may define a cut off for consideration for a given competition cycle, but this should be disputable by the core team.

Reply:
As stated before, the _Dev Team_ will consult the _Core Team_, but as an independent group, they are free to ignore them.

Reply:
Also note that as the Trustees ultimately approve the budget, they act as checks and balances.

The Dev Team are free to ignore **this year's** Core Team, and should have that right as their responsibility extends beyond that of the Core Team and so the interests of the two may be fundamentally different.

However, if the Dev Team decides to ignore everyone and just develop lots of pink machine-learning unicorns, it is unlikely they will be approved budget for this endeavor. This ensures that the Dev Team maintains freedom in development while also ensuring they can't do things which are plain stupid.

Reply:
I think my original wording is a little off. Yes the dev team can consult the core team for a cut off date for a given project, but as the core team have the final say as to whether something ships in a given competition cycle, they may disagree with the given cut-off date.

Reply:
The Core Team ultimately makes the decision on whether something is shipped as stated elsewhere. I don't see any other issues here.